### ISSUE TITLE
More false positive for unused imports

### ISSUE THREAD

**Issue by eskatos** [arturbosch/detekt#457]
```kotlin
package com.acme

import com.acme.cathedral.TheDome   // here
import com.acme.bazar.SomeException // and here

/**
 * Do something.
 * @throws SomeException when ...
 * @see TheDome.someMethod
 */
fun doSomething() {
    TODO()
}
```

Looks related to #201

**Comment by arturbosch** [comment]
Thanks for the reporting!
Will look at this issues asap.

**Comment by Mauin** [comment]
There seems to be 2 issues here:

First: Detekt doesn't account for Tags in KDoc at the moment. I'm looking into a fix right now.
Second: If you reference classes from KDoc you should put them in square brackets for detekt to pick them up. (see also: https://kotlinlang.org/docs/reference/kotlin-doc.html#linking-to-elements)
```
/**
 * Do something.
 * @throws [SomeException] when ...
 * @see [TheDome.someMethod]
 */
```

**Comment by Mauin** [comment]
I put #458 together which should solve the issue. However you would still need to correctly link to the classes with `[SomeException]`. I took over the code from this issue as a test case for the rule with the added square braces. 

https://github.com/arturbosch/detekt/pull/458/files#diff-80fb3301450b6c6c9842db6565cef77dR123

**Comment by eskatos** [comment]
Wow that was fast, thanks!
Right, this made me realize that the square braces are required for dokka to generate proper links.
That's perfect!
